boxnav/boxnav/boxnavigator.py

```diff
@@ -363,3 +363,50 @@ def __update_target_if_necessary(self) -> None:
             # self.dominant_direction = self.determine_direction_to_target(self.target)
             # self.anchor_1 = self.rotation_anchor(self.target, self.current_box)[0]
             # self.anchor_2 = self.rotation_anchor(self.target, self.current_box)[1]
+
+        # TODO: for Tommy
+        # Maybe also pass in the current box from navigator? (I like this more as of now)
+        # index = self.boxes.index(self.get_boxes_enclosing_point(pt)[0])
+        # index = self.boxes.index(current_box)
+        # Or maybe the index of the current box?
+
+    def get_percent_through_env(self, pt: Pt, initial_pt: Pt, boxes: list) -> float:
+        """
+        takes the current and initial location of an agent, and its current environment,
+        returns its % completion of said environment
+
+        :param: pt (class Pt) updated location of agent
+        :param: initial_pt (class Pt) starting location of agent
+        :param: boxes (list) selected environment from boxnav.envrionments
+        :return: (float) percent completion of environment
+        """
```

Comment:
Normally I'd be in favor of added documentation, but in this case I think the parameters and use of the function are obvious enough that we don't need this docstring.